reference/verb/edit.rst

```diff
@@ -0,0 +1,50 @@
+``edit`` - Edit File
+====================
+
+The ``edit`` search and edit a file inside a package using command line.
+It is provided by the ``colcon-ed`` package.
+
+When a correct package name and file name is provided, it will open that
+file with an editor. 
+
+.. code-block:: bash
+
+    $ colcon edit <package_name> <file_name>
+
+.. note::
+
+    To enable auto-complete, check out the 
+    :doc:`installation guide <../../user/installation>`.
+
+Hidden files and ``.pyc`` files are ignored. If multiple files under 
```

Comment:
The docs aim to follow [one-sentence-per-line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) which is unfortunately not covered by the linter atm. Please start each sentence on a new line.

Reply:
Thanks. Changed as requested.